python/Ganga/Core/GangaRepository/GangaRepositorySQLite.py

```diff
@@ -34,7 +34,7 @@ def startup(self):
             os.makedirs(self.root)
         except OSError as x:
             pass
-        self.con = sqlite.connect(os.path.join(self.root, "database.db"))
+        self.con = sqlite3.connect(os.path.join(self.root, "database.db"))
```

Comment:
Are we ever using this backend? Should we really be touching code we will likely drop if not?